pipeline_dp/budget_accounting.py

```diff
@@ -458,12 +473,12 @@ def __init__(self,
                               "NaiveBudgetAccountant instead of "
                               "PLDBudgetAccountant")
 
-        self.minimum_noise_std = None
+        self.base_noise_std = None
```

Comment:
what's the difference between minimum and base? should we add a comment highlighting that it is base and not minimum?